cms/drupal/db/backup/create.php

```diff
@@ -28,10 +28,11 @@
     // Create the backup files.
     $appPath = get('app_path');
     $timeout = get('db_backup_create_timeout', 60 * 60);
-    $sites = get('sites');
-    if (!is_array($sites)) {
-        $sites = explode(',', $sites);
+    $sites   = get('sites');
+    if (! \is_array($sites)) {
+        $sites = \explode(',', $sites);
     }
+
     foreach ($sites as $site) {
         within($appPath . '/sites/' . $site, static function () use ($site, $timeout): void {
             $command = \vsprintf('{{drush}} sql:dump --gzip --result-file={{backups}}/{{namespace}}-{{project}}-%s-%s.sql', [
```

Comment:
I'd love it if we could add the server name into the db backup file name here.  So if we ever have those files side by side we'll be able to tell where they came from (staging vs production).

Reply:
I agree, though I think we should address that in a follow-up release. I'll make sure there's a todo item for that...